src/js/components/AppPageComponent.jsx

```diff
@@ -182,6 +182,26 @@ var AppPageComponent = React.createClass({
     }
   },
 
+  getUnhealthyTaskMessage: function (healthCheckResults = []) {
+    return healthCheckResults.map((hc, index) => {
+      if (hc && !hc.alive) {
+        var failedCheck = this.state.app.healthChecks[index];
+
+        var protocol = failedCheck.protocol
+          ? `${failedCheck.protocol} `
+          : "";
```

Comment:
I think we should have a conversation on this. I'm pretty sure @mlunoe has some strong feelings on this as well, if I remember correctly. Could we hold on using ternary operators until we come to a consensus?